### PR TITLE
RK-286 added back link, cookie concent, language toggle and tech supp…

### PR DESCRIPTION
**Pull request by shiromi**
…ort feature on template

### CODE REVIEW

File: test/uk/gov/hmrc/taxfraudreportingfrontend/controllers/ViewUtilsSpec.scala [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  def messages(app: Application): Messages = app.injector.instanceOf[MessagesApi].preferred(FakeRequest())
+
+  "ViewUtils" should {
+    "titleNoForm should produce a title with no form" in {
+      val application = GuiceApplicationBuilder().build()
+      ViewUtils.titleNoForm("testTime", None)(
+        messages(application)
+      ) mustBe "testTime - Report tax fraud or evasion - GOV.UK"
+    }
+    /*
```
Comment (shinealex33) [inline test/uk/gov/hmrc/taxfraudreportingfrontend/controllers/ViewUtilsSpec.scala:43]: Please remove the commented code if it is not required.

File: project/plugins.sbt [hunk 2]
```diff
@@ -9,3 +9,6 @@ addSbtPlugin("com.typesafe.sbt"  % "sbt-gzip"           % "1.0.2")
 addSbtPlugin("org.irundaia.sbt"  % "sbt-sassify"        % "1.5.1")
 addSbtPlugin("org.scalameta"     % "sbt-scalafmt"       % "2.4.2")
 addSbtPlugin("org.scoverage"     % "sbt-scoverage"      % "1.7.0")
+addSbtPlugin("net.ground5hark.sbt" % "sbt-concat" % "0.2.0")
+addSbtPlugin("com.typesafe.sbt" % "sbt-uglify" % "2.0.0")
+addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")
```
Comment (shinealex33) [inline project/plugins.sbt:14]: add end of the line

File: conf/messages [hunk 3]
```diff
@@ -1,2 +1,10 @@
...
+service.name = Report tax fraud or evasion
+site.govuk = GOV.UK
 service.text = This is your new service
+error.title = Error
+index.title = Using this service
+index.header = Using this service
+site.back = Back
+
+error.browser.title.prefix = Error:
+error.summary.title = There is a problem
```
Comment (shinealex33) [inline conf/messages:10]: add end of the line

File: conf/app.routes [hunk 4]
```diff
@@ -1,4 +1,5 @@
 ->         /hmrc-frontend                                hmrcfrontend.Routes
 GET        /assets/*file                                 controllers.Assets.versioned(path = "/public", file: Asset)
+GET        /language/:lang                               @uk.gov.hmrc.taxfraudreportingfrontend.controllers.LanguageSwitchController.switchToLanguage(lang: String)
 
-GET        /helloWorld                              @uk.gov.hmrc.taxfraudreportingfrontend.controllers.HelloWorldController.helloWorld
\ No newline at end of file
+GET        /helloWorld                                   @uk.gov.hmrc.taxfraudreportingfrontend.controllers.HelloWorldController.helloWorld
```
Comment (shinealex33) [inline conf/app.routes:5]: please add end of the line

File: app/uk/gov/hmrc/taxfraudreportingfrontend/views/components/error_summary.scala.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                }else {
+                <a href="#@{error.key}">@messages(error.message, error.args:_*)</a>
+                }
+            </li>
+            }
+        </ul>
+    </div>
+</div>
+
+}
```
Comment (shinealex33) [inline app/uk/gov/hmrc/taxfraudreportingfrontend/views/components/error_summary.scala.html:47]: please add end of the line

File: app/uk/gov/hmrc/taxfraudreportingfrontend/views/components/errorSummary.scala.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+        href = Some(s"#$id"),
+        content = Text(messages(error.message, error.args:_*))
+        )
+    }) { errorLinks =>
+        @govukErrorSummary(ErrorSummary(
+        errorList = errorLinks,
+        title = Text(messages("error.summary.title"))
+    ))
+    }
+}
```
Comment (shinealex33) [inline app/uk/gov/hmrc/taxfraudreportingfrontend/views/components/errorSummary.scala.html:49]: please add end of the line

File: app/uk/gov/hmrc/taxfraudreportingfrontend/views/ViewUtils.scala [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+  def title(form: Form[_], titleStr: String, section: Option[String] = None, titleMessageArgs: Seq[String] = Seq())(
+    implicit messages: Messages
+  ): String =
+    titleNoForm(s"${errorPrefix(form)} ${messages(titleStr, titleMessageArgs: _*)}", section)
+   */
+  def titleNoForm(title: String, section: Option[String] = None)(implicit messages: Messages): String =
+    s"${messages(title)} - ${section.fold("")(messages(_) + " - ")}${messages("service.name")} - ${messages("site.govuk")}"
+
+  /*
+  def errorPrefix(form: Form[_])(implicit messages: Messages): String =
```
Comment (shinealex33) [inline app/uk/gov/hmrc/taxfraudreportingfrontend/views/ViewUtils.scala:34]: How do we handle the error Title. We don't want to merge the commented code into main

File: app/uk/gov/hmrc/taxfraudreportingfrontend/views/ViewUtils.scala [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+ */
+
+package views
+
+import play.api.data.Form
+import play.api.i18n.Messages
+
+object ViewUtils {
+
+  /*
```
Comment (shinealex33) [inline app/uk/gov/hmrc/taxfraudreportingfrontend/views/ViewUtils.scala:24]: why we commented this piece of this code

File: app/uk/gov/hmrc/taxfraudreportingfrontend/views/LanguageSelect.scala.html [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+@this(hmrcLanguageSelect: HmrcLanguageSelect)
+
+@()(implicit messages: Messages, appConfig: AppConfig)
+
+@hmrcLanguageSelect(LanguageSelect(
+    if (messages.lang.code == "cy") Cy else En,
+    (En, switchToLanguage("en").url),
+    (Cy, switchToLanguage("cy").url)
+))
```
Comment (shinealex33) [inline app/uk/gov/hmrc/taxfraudreportingfrontend/views/LanguageSelect.scala.html:30]: add end of the file

**Review by shinealex33 — CHANGES_REQUESTED** [review]
please fix the review comments

File: app/uk/gov/hmrc/taxfraudreportingfrontend/views/ViewUtils.scala (hunk 7 above)
Comment (shiromi) [reply app/uk/gov/hmrc/taxfraudreportingfrontend/views/ViewUtils.scala:34]: once we add the error will add the code back. for now I have removed the commented lines.

**Review by shinealex33 — APPROVED** [review]
LGTM

**Pull request merged**